src/platformIcon.tsx

```diff
@@ -160,12 +160,14 @@ const PlatformIcon = ({
       <div {...otherProps} style={{ position: "relative", ...style }}>
         <img
           src={iconPath}
+          className="platformicon"
           width={size}
           height={size}
           style={{ borderRadius: `${radius}px` }}
         />
         <img
           src={languageIconPath}
+          className="platformicon"
```

Comment:
You may only need to apply className to the one below if you're using the platform logos without the language icon in the bottom right corner. Otherwise, this may cause a conflict.

Reply:
Good point!